target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Wed Jul 08 11:09:27 AST 2020
+#Wed Jul 08 17:19:44 AST 2020
 groupId=com.google.sps
 artifactId=portfolio
```

Comment:
You should have a different artifactId for this new project?

Reply:
I did not touch that, because the last time when I did, it was not running and I was getting error 403. I am not sure how to go about changing this. I tried before and it did not work.